src/Reflection/ReflectionClass.php

```diff
@@ -332,6 +326,7 @@ private function createMethodsFromTrait(ReflectionMethod $method): array
 
         if (array_key_exists($methodHash, $this->traitsData['modifiers'])) {
             // PhpParser modifiers are compatible with PHP reflection modifiers
+            /** @var int-mask-of<ReflectionMethodAdapter::IS_*> $methodModifiers */
```

Comment:
Can this not be inferred? 🤔

Reply:
Cherry-picked two commits from my next PR.

Psalm can infer the type now, PHPStan does not.
https://github.com/Roave/BetterReflection/pull/1236/commits/8f89aae63ac23aa1b5aa85d881db7f5b32fdf467